client/state/ui/README.md

```diff
@@ -2,39 +2,3 @@ UI State
 ========
 
 A module for managing user interface state.
-
-## Actions
```

Comment:
:) I'm fine with it as long as it's generated from the source files. Manually updated files drift pretty quickly. There should be a number of options to parse jsdoc and generate markdown. We can add it as a build step.